### DISCUSSION TITLE
Sub-button visibility

### DISCUSSION THREAD

**Discussion by alrhalford** [Clooos/Bubble-Card#787] [Feature requests]
I have copied the button that triggers a pop up example, and it looks great. Sometimes, it would be useful to have a sub button to show the state of some entity, but only visible depending on the state.

For example, I have a pop-up for everything to do with my cat, and button that triggers it called 'Cat'. It would be useful to see at a glance if the cat has used the litterbox, so it would be great to have a sub-button (or some kind of display, it doesn't necessarily have to be click-able) that is visible if the cat has used the litterbox, and not visible if not. I have attached an example:
![image](https://github.com/user-attachments/assets/800d69c7-9445-43ae-8252-8b5a2202e1f5)

I know it is possible to change that icon based on the entity's state, using custom styles, but I would love to be able to hide it completely.

**Comment by Clooos** [comment]
Hi, this is already possible, take a look at the templates section in the documentation, I've added this exact example 🙂

**Reply by alrhalford** [reply]
I totally missed this, thank you, and thanks for all your hard work!

**Reply by Clooos** [reply]
You're welcome!
